deploy-chroot/reinstall-apt-setup.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e -u
+
+sudo apt-get -y purge homeworld-apt-setup && sudo apt-get -y remove 'homeworld-*' && sudo apt-get clean || true
```

Comment:
What is the purpose of the '|| true' here? Wouldn't we want to know if one of these three steps were to fail?

Reply:
I do not remember why that is there

Reply:
Oh, it's so that this works the first time you run the script.

Reply:
Do we need it on the `apt-get clean` command?

Reply:
If the `apt-get remove` fails then we don't want to run `apt-get clean` at all.

Reply:
Hmm, okay.